### ISSUE TITLE
Elmish 0.9 don't use the same code as in the repo

### ISSUE THREAD

**Issue by MangelMaxime** [fable-elmish/elmish#104]
### Description

When trying to use Elmish 0.9 we don't have the same as in the github repo causing this error: 

`packages\Fable.Elmish\fable\program.fs(67,12): (67,70) error FSHARP: The member or object constructor 'WriteLine' does not take 2 argument(s). An overload was found taking 0 arguments.`

We have this code in the Nuget code: 

```fs
    let withConsoleTrace (program: Program<'arg, 'model, 'msg, 'view>) =
        let inline toPlain o = toJson o |> JS.JSON.parse
        let traceInit (arg:'arg) =
            let initModel,cmd = program.init arg
            System.Console.WriteLine ("Intial model: {0}", toPlain initModel)
            initModel,cmd

        let traceUpdate msg model =
            System.Console.WriteLine("Processing: {0}", toPlain msg)
            let newModel,cmd = program.update msg model
            System.Console.WriteLine("Updated: {0}", toPlain newModel)
            newModel,cmd

        { program with
            init = traceInit 
            update = traceUpdate }
```

where we should have:

```fs
    let withConsoleTrace (program: Program<'arg, 'model, 'msg, 'view>) =
        let inline toPlain o = toJson o |> JSON.parse
        let traceInit (arg:'arg) =
            let initModel,cmd = program.init arg
            console.log ("New model:", toPlain initModel)
            initModel,cmd

        let traceUpdate msg model =
            console.log ("New message:", toPlain msg)
            let newModel,cmd = program.update msg model
            console.log ("Updated:", toPlain newModel)
            newModel,cmd

        { program with
            init = traceInit 
            update = traceUpdate }
```

@et1975 Can you republish a 0.9.1 version ?

**Comment by mike-morr** [comment]
I am getting the same error after Paket update.

**Comment by alfonsogarciacaro** [comment]
Sorry, this is my fault, apologies 😭 

You can fix it by upgrading both Fable.Core and dotnet-fable to 1.1.3.

**Comment by mike-morr** [comment]
Confirmed, updating to 1.1.3 resolves the issue.

**Comment by et1975** [comment]
Nah, it's my fault, bug in the build script - wasn't running the build as part of Publish target, sorry.
Released the fix in 0.9.1.

**Comment by MangelMaxime** [comment]
Ok so I say it's both your fault 👿 😄 

So now everything is running fine: thanks to both of you 👍
